docker/cassandra-harry/README.md

```diff
@@ -0,0 +1,6 @@
+```
+export HARRY_DOCKER_IMAGE=scylladb/hydra-loaders:cassandra-harry-jdk11-$(date +'%Y%m%d')
```

Comment:
maybe tag by cassandra harry commit id instead of date? Or both maybe?

Reply:
we check it out inside the Dockerfile, so for doing that you need to checkout the code outside the docker build...
since we are not changing this tool too much, date is good enough.